web/src/lib/components/admin-page/restore-dialogue.svelte

```diff
@@ -36,6 +37,10 @@
   onCancel={() => dispatch('cancel')}
 >
   <svelte:fragment slot="prompt">
-    <p><b>{user.name}</b>'s account will be restored.</p>
+    <p>
+      <FormatMessage message={$json('admin.user_restore_description')} values={{ user: user.name }} let:message>
+        <b>{message}</b>
```

Comment:
In this case the translation string also has `<b></b>`. Do we need both?

Reply:
HTML from translations is never rendered directly to mitigate xss risk. So yes, we need both.